section4/exercises/classMethods.js

```diff
@@ -27,14 +27,38 @@ Make sure to run the file with node in your command line.
 // Call the `addTopping` method on two burritos, then log the burritos to verify
 // that the proteins have been changed.
 
+var  baseToppings = ['white rice', 'black beans', 'lettuce', 'tomato', 'onion', 'cheese', 'sour cream'];
+
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
+  }
+
+  changeProtein() {
```

Comment:
The method `changeProtein` should accept a parameter (see instructions) and assign that parameter to the `this.protein` variable.